src/ontology/mondo-edit.obo

```diff
@@ -434087,6 +434069,45 @@ intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc
 relationship: has_modifier HP:0000007 {source="OMIM:606176"} ! Autosomal recessive inheritance
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100170
+name: intellectual disability-developmental delay-contractures syndrome
```

Comment:
I think we should be consistent with naming conventions, probably best to choose OMIM here as we use OMIM for subclasses